cypress.config.js

```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
```

Comment:
could we move that to it's own folder packages?
The vision on the repo is to have everything under packages (even front and server but that's another story)